client/Components/Games/NewGame.jsx

```diff
@@ -36,6 +36,7 @@ class NewGame extends React.Component {
             password: '',
             useGameTimeLimit: false,
             hideDecklists: false,
+            adaptive: { chains: 0, selection: [], biddingWinner: '', match1Winner: '', match2Winner: '', match3Winner: '' },
```

Comment:
As per the other PR, default this on the server, don't send it across from the client